Shaders/ssr_pass/ssr_pass.frag.glsl

```diff
@@ -45,28 +44,27 @@ float getDeltaDepth(const vec3 hit) {
 
 vec4 binarySearch(vec3 dir) {
 	float ddepth;
-	vec3 start = hitCoord;
-	for (int i = 0; i < numBinarySearchSteps; i++) {
+	for (int i = 0; i < 7; i++) {
```

Comment:
Should it be `i < maxSteps` instead of `i < 7`?

Reply:
No, maxSteps are used for rayCast, binary search narrows down the output of this function and the original ssr shader sets a ssrMinRayStep to 7.